lib/ClimaLandSimulations/src/Fluxnet/fluxnet_sites/US-Ha1/US-Ha1_parameters.jl

```diff
@@ -0,0 +1,126 @@
+export harvard_default_params
+
+"""
+Site-specific model parameters for running CliMA LSM on the Harvard
+fluxtower site.
+"""
+harvard_default_params = (
+    hetero_resp = (
+        ## LAI data from MODIS
+        # Heterotrophic respiration parameters
+        θ_a100 = FT(0.1816),
+        D_ref = FT(1.39e-5),
+        b = FT(4.547),
+        D_liq = FT(3.17),
+        # DAMM
+        α_sx = FT(194e3),
+        Ea_sx = FT(61e3),
+        kM_sx = FT(5e-3),
+        kM_o2 = FT(0.004),
+        O2_a = FT(0.209),
+        D_oa = FT(1.67),
+        p_sx = FT(0.024),
+    ),
+    auto_resp = (
+        # Autotrophic respiration parameters
+        ne = FT(8 * 1e-4),
+        ηsl = FT(0.01),
+        σl = FT(0.05),
+        μr = FT(1.0),
+        μs = FT(0.1),
+        f1 = FT(0.012),
+        f2 = FT(0.25),
+    ),
+    soil = (
```

Comment:
soil and soil heat should be combined [later PR ok]

Reply:
(parameters in soil_heat also greatly affect soil moisture, they arent really separate models)

Reply:
or we can discuss other ways to split - e.g. the roughness lengths are also used for evaporation

Reply:
I modified the site params to match ozark, which was the only one up to date (sorry!)